app/design/frontend/Magento/luma/web/css/source/_forms.less

```diff
@@ -91,11 +91,7 @@
     .select-styling() {
         .lib-css(appearance, none, 1);
         appearance: none;
-        background: @select__background url('../images/select-bg.svg') no-repeat 100% 45%;
         background-size: 30px 60px;
-        border: 1px solid @border-color__base;
```

Comment:
Where are the @select__border and other variables defined / used?

Reply:
It's define in lib/web/css/source/lib/variables/_forms.less

Reply:
But I do not (directly) see its usage anywhere. I'm speaking about the newly added variables.

Reply:
I didn't created/added any new variables. all variables are in lib/web/css/source/lib/variables/_forms.less

Reply:
Ah, found them =) `@{@{_type}__background}` and so on.

Reply:
Sorry, just want to check the inheritance and usage of the variables to understand the issue better and detect similar issues.